### DISCUSSION TITLE
Não envia e-mail

### DISCUSSION THREAD

**Discussion by adeiltonpessini** [RamonSilva20/mapos#2260] [General]
Boa tarde, alguém conseguiria me ajudar, estou tentando configurar o SMTP, mas não envia de jeito nenhum.
Já fiz os Cron's, ja configurei o arquivo email.php.
irei deixar uns prints abaixo!
Cron: https://prnt.sc/vhXJDgvtHHRO
email.php: https://prnt.sc/P3kUjXovet6Z

**Comment by Pr3d4dor** [comment]
@adeiltonpessini verifica se sua hospedagem não está bloqueando.

**Reply by adeiltonpessini** [reply]
Já fiz de tudo, mas não vai.
Se eu mudar de TLS para SSL, apresenta falha no envio. Se eu colocar TLS fica enviando e nunca envia...
E apresenta esse erro no log

</pre><pre>from: 250 OK
</pre><pre>to: 250 Accepted
</pre><pre>data: 354 Enter message, ending with "." on a line by itself
</pre>550-Your FROM address ( noreply@jpeletroetecnologia.com.br,
550-jonair@jpeletroetecnologia.com.br ) must match your authenticated email
550-user ( noreply@jpeletroetecnologia.com.br ). Treating this as a spoofed
550 email.
<br /><pre>quit: 221 rac.boxsecured.com closing connection
</pre>The following SMTP error was encountered: 550-Your FROM address ( noreply@jpeletroetecnologia.com.br,
550-jonair@jpeletroetecnologia.com.br ) must match your authenticated email
550-user ( noreply@jpeletroetecnologia.com.br ). Treating this as a spoofed
550 email.
<br />Unable to send email using PHP SMTP. Your server might not be configured to send mail using this method.

**Comment by paulosardinha** [comment]
Tenho o mesmo problema. Não muda o status de Pendente nunca. Não consigo configurar de maneira alguma, tem algo diferente que possa fazer?
